README.md

```diff
@@ -148,82 +155,83 @@ This feature is still under development, review, and testing.
     my-binding   Opaque   5      32s
     ```
     
-    See [Using Secrets](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets) for the different options to use the credentials from your application running in the Kubernetes cluster, 
+    See [Using Secrets](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets) to learn about different options on how to use the credentials from your application running in the Kubernetes cluster, 
 
 [Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
 
 ## Reference documentation
 
 ### Service Instance
 #### Spec
-| Property         | Type     | Comments                                                                                                   |
+| Property         | Type     | Description                                                                                                   |
 |:-----------------|:---------|:-----------------------------------------------------------------------------------------------------------|
-| serviceOfferingName`*`   | `string`   | The SAP BTP service offering name |
-| servicePlanName`*` | `string`   |  The plan to use for the service instance |
-| servicePlanID   |  `string`   |  The plan ID in case service offering and plan name are ambiguous |
-| externalName       | `string`   |  The name for the service instance in SAP BTP, defaults to the binding `metadata.name` if not specified |
-| parameters       |  `[]object`  |  Provisioning parameters for the instance, check the documentation of the specific service you are using for details |
+| serviceOfferingName`*` | `string`  | The name of the SAP BTP service offering. |
+| servicePlanName`*` | `string`   |  The plan to use for the service instance. |
+| servicePlanID   |  `string`   |  The plan ID in case service offering and plan name are ambiguous. |
+| externalName       | `string`   |  The name for the service instance in SAP BTP, defaults to the instance `metadata.name` if not specified. |
+| parameters       |  `[]object`  |  Some services support the provisioning of additional configuration parameters during the instance creation. For the list of supported                                  parameters, check the documentation of the particular service offering.|
```

Comment:
can you remove the whitespaces? same for the other descriptions in the tables below